packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -611,8 +618,12 @@ function logFileStream(fileStream) {
 }
 
 async function getFiles(folderPath) {
-  const dir = await cozy.files.statByPath(folderPath)
-  const files = await queryAll('io.cozy.files', { dir_id: dir._id })
+  const dir = await client.collection(FILES_DOCTYPE).statByPath(folderPath)
```

Comment:
Are we sure the `dir` is always defined here? It might be worth adding some existing check on `dir.data._id`?

Reply:
statByPath throw an error if there is no file. So checking the dir.data._id doesn't make sense in that case.